tests/ui/traits/const-traits/trait-default-body-stability.rs

```diff
@@ -1,4 +1,5 @@
-//@ known-bug: #110395
+//@ run-pass
+// @ known-bug: #110395
```

Comment:
```suggestion
//! This test used to ICE(#110395), because....
```

add some explanation by looking at the ICE issue's backtrace and ICE message

Reply:
oh lol, I thought it was an ICE or sth, yea just remove the known-bug line and use `check-pass`